docs/guide/http_cache.md

```diff
@@ -318,6 +318,10 @@ fastcgi_param SYMFONY_HTTP_CACHE 0;
 fastcgi_param SYMFONY_TRUSTED_PROXIES "193.22.44.22";
 ```
 
+!!! caution "Trusted proxies when using SSL offloader / loadbalancer"
```

Comment:
`Tusted proxies when using SSL proxy / loadbalancer in combination with Varnish`